### PR TITLE
Improved typedoc page

### PR DESCRIPTION
**Pull request by edgardmessias**
This PR improve the doc page, removindo duplicate entrys.

image

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -14,6 +14,6 @@
     "declaration": true,
     "lib": ["es2020", "esnext.array"]
   },
-  "include": ["src/*/*.ts"],
+  "include": ["src/**/*.ts"],
```
Comment (adiwajshing) [inline tsconfig.json:17]: Why is this here?

File: src/typedocEntry.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export * from './../WAMessage/WAMessage'
```
Comment (adiwajshing) [inline src/typedocEntry.ts:1]: Doesn't `src/WAConnection/WAConnection.ts` work instead of this?

**Comment by adiwajshing** [comment]
@edgardmessias PR is much appreciated. Have added a few questions.

File: tsconfig.json (hunk 1 above)
Comment (edgardmessias) [reply tsconfig.json:17]: * is one or more subfolders
** is zero, one or more subfolders

It is necessary to include root files to compile, like src/<name>.ts

File: src/typedocEntry.ts (hunk 2 above)
Comment (edgardmessias) [reply src/typedocEntry.ts:1]: No, because it will generate docs only of exported methods, classes and constants.

The beta version of typedoc works like a library instead of module and each entry is a module, so to generate a single library, we must create a single file with all exportable content.

The idea is the same of files entry in package.json

File: tsconfig.json [hunk 3]
```diff
@@ -14,6 +14,6 @@
...
   "exclude": ["node_modules", "src/Tests/*", "src/Binary/GenerateStatics.ts"]
```
Comment (adiwajshing) [inline tsconfig.json:18]: Should probably exclude `typedocentry.ts`

Comment (edgardmessias) [reply tsconfig.json:18]: No, because typedoc use the typescript compiler to generate the docs.

As a alternative, we can rename the `typedocentry.ts` to `index.ts` and edit the files of package.json from `"files": ["lib/WAConnection/*","lib/Binary/*","WAMessage/*"]` to `"files": ["lib/index.js"]`

Comment (edgardmessias) [reply tsconfig.json:18]: The ideia is to show what the `@adiwajshing/baileys` exports

Comment (adiwajshing) [reply tsconfig.json:18]: Yeah I'm good with that

Comment (edgardmessias) [reply tsconfig.json:18]: So, @adiwajshing , Would you like me to make these changes and update this PR?

Comment (adiwajshing) [reply tsconfig.json:18]: yes please

Comment (edgardmessias) [reply tsconfig.json:18]: I did

**Comment by adiwajshing** [comment]
@edgardmessias looks good. Just resolve the conflict & I'll merge.

**Comment by edgardmessias** [comment]
@adiwajshing  Done
